docs/XSLT-Modul-Dokumentation.md

```
@@ -348,16 +388,31 @@ Rollen skapas automatiskt i LDAP vid ETERNA:s första start (bootstrap) baserat
 
 ## REST API
 
+### Lista tillgängliga stilmallar
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)